### DISCUSSION TITLE
Disable Admin Panel via addon config

### DISCUSSION THREAD

**Discussion by beMoD** [hassio-addons/addon-bitwarden#270] [Ideas]
Hi,

would it be possible to add a configuration in the addon to disable the admin panel? Right now I am disabling it by renaming the config.json file in the file system and I am aware this is neither supported nor ideal. Would be great to be able to disable it just directly through the addons config.

**Comment by Rohrleitner** [comment]
where can i find vaultwarden in the file system to also do this?  In addon_configs is no vaultwarden!

**Reply by beMoD** [reply]
Inside the docker container. So you need to exec into it and edit it there.
